command.go

```diff
@@ -286,8 +285,7 @@ func (c *Command) HelpTemplate() string {
 		return c.parent.HelpTemplate()
 	} else {
 		return `{{with or .Long .Short }}{{. | trim}}{{end}}
-{{if or .Runnable .HasSubCommands}}{{.UsageString}}{{end}}
-`
```

Comment:
The extra line after the "Flags:" section I don't think I can reasonably fix until https://github.com/golang/go/issues/9969  which is golang 1.6   :-(